### PR TITLE
Upgrade to JmDNS 3.5.12

### PR DESCRIPTION
**Pull request by kaikreuzer**
See [changelog 3.5.10](https://github.com/jmdns/jmdns/releases/tag/3.5.10) (and [changelog 3.5.11](https://github.com/jmdns/jmdns/releases/tag/3.5.11) and [changelog 3.5.12](https://github.com/jmdns/jmdns/releases/tag/3.5.12))

### CODE REVIEW

**Review by wborn — CHANGES_REQUESTED** [review]
I think the Karaf feature still needs to be updated.

**Comment by kaikreuzer** [comment]
Hm, I thought I did it this time, but it is indeed missing in the commit...
Anyhow, let's wait with the merge until https://github.com/jmdns/jmdns/issues/281 is clarified.

File: bom/compile/pom.xml [hunk 1]
```diff
@@ -195,7 +195,7 @@
     <dependency>
       <groupId>org.jmdns</groupId>
       <artifactId>jmdns</artifactId>
-      <version>3.5.9</version>
+      <version>3.5.11</version>
```
Comment (holgerfriedrich) [inline bom/compile/pom.xml:198]:
```suggestion
      <version>3.5.12</version>
```

File: bom/runtime/pom.xml [hunk 2]
```diff
@@ -467,7 +467,7 @@
     <dependency>
       <groupId>org.jmdns</groupId>
       <artifactId>jmdns</artifactId>
-      <version>3.5.9</version>
+      <version>3.5.11</version>
```
Comment (holgerfriedrich) [inline bom/runtime/pom.xml:470]:
```suggestion
      <version>3.5.12</version>
```

File: features/karaf/openhab-tp/src/main/feature/feature.xml [hunk 3]
```diff
@@ -183,8 +183,8 @@
 	</feature>
 
 	<feature name="openhab.tp-jmdns" description="An implementation of multi-cast DNS in Java." version="${project.version}">
-		<capability>openhab.tp;feature=jmdns;version=3.5.9</capability>
-		<bundle>mvn:org.jmdns/jmdns/3.5.9</bundle>
+		<capability>openhab.tp;feature=jmdns;version=3.5.11</capability>
+		<bundle>mvn:org.jmdns/jmdns/3.5.11</bundle>
```
Comment (holgerfriedrich) [inline features/karaf/openhab-tp/src/main/feature/feature.xml:187]:
```suggestion
		<capability>openhab.tp;feature=jmdns;version=3.5.12</capability>
		<bundle>mvn:org.jmdns/jmdns/3.5.12</bundle>
```

File: itests/org.openhab.core.config.discovery.mdns.tests/itest.bndrun [hunk 4]
```diff
@@ -19,7 +19,7 @@ Fragment-Host: org.openhab.core.config.discovery.mdns
 	si-units;version='[2.1.0,2.1.1)',\
 	si.uom.si-quantity;version='[2.1.0,2.1.1)',\
 	org.osgi.util.function;version='[1.2.0,1.2.1)',\
-	javax.jmdns;version='[3.5.9,3.5.10)',\
+	javax.jmdns;version='[3.5.11,3.5.12)',\
```
Comment (holgerfriedrich) [inline itests/org.openhab.core.config.discovery.mdns.tests/itest.bndrun:22]:
```suggestion
	javax.jmdns;version='[3.5.12,3.5.13)',\
```

**Review by holgerfriedrich — CHANGES_REQUESTED** [review]
Ready to switch to 3.5.12?

**Comment by kaikreuzer** [comment]
> Ready to switch to 3.5.12?

For sure!
But to be on the safe side, I would suggest waiting with merging it until after 4.3.0.M1.

**Review by holgerfriedrich — APPROVED** [review]
LGTM

**Comment by holgerfriedrich** [comment]
@wborn I think your review comment has been addressed, can we merge?

**Review by wborn — APPROVED** [review]
Thank you both!

**Pull request merged**